### PR TITLE
rebased and added ImageWidgetCrop test in the existing test file

### PR DESCRIPTION
**Pull request by tduong992**
Re-create PR. From [d.o](https://www.drupal.org/node/2579519).

### CODE REVIEW

File: src/Tests/ImageWidgetCropTest.php [hunk 1]
```diff
@@ -88,6 +88,66 @@ public function testCropUi() {
...
+
+    // Assert that there is no crop widget, neither 'Alternative text' text
+    // filed nor 'Remove' button yet.
+    $raw = '<summary role="button" aria-controls="edit-field-image-crop-test-0-crop-preview-wrapper" aria-expanded="false" aria-pressed="false">Crop image</summary>';
+    $this->assertNoRaw($raw);
+    $this->assertNoText('Alternative text');
+    $this->assertNoFieldByName('field_image_crop_test_0_remove_button');
+
+    // Upload an image in field_image_crop_test_0.
+    $image['files[field_image_crop_test_0]'] = \Drupal::service('file_system')->realpath('public://image-test.jpg');
```
Comment (slashrsm) [inline src/Tests/ImageWidgetCropTest.php]: I'd rather use $this->container than \Drupal

File: src/Tests/ImageWidgetCropTest.php [hunk 2]
```diff
@@ -88,6 +88,66 @@ public function testCropUi() {
...
+    $this->assertFieldByName('field_image_crop_test_0_remove_button');
+
+    // Set title and 'Alternative text' text field and save.
+    $title = $this->randomMachineName();
+    $edit = array(
+      'title[0][value]' => $title,
+      'field_image_crop_test[0][alt]' => $this->randomMachineName(),
+    );
+    $this->drupalPostForm(NULL, $edit, 'Save');
+    $this->assertText('Crop test '. $title . ' has been created.');
```
Comment (slashrsm) [inline src/Tests/ImageWidgetCropTest.php]: Space missing before first .

**Comment by slashrsm** [comment]
Please reword your commit message to be of standard Drupal form (reference to issue # and contributors).

**Comment by woprrr** [comment]
It's ok for me :) Great job. thanks a lot @tduong992

**Pull request merged**